README.md

```diff
@@ -1,25 +1,50 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# React Todo App
 
-## Available Scripts
+A React App that keeps track of your tasks and lets you add, edit, mark as completed and delete them.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- React
+
+## Getting Started
+
+To get a local copy, run the following command.
+
+- `git clone git@github.com:jfoyarzo/react-todo-app.git`
+
+### Prerequisites
+
+- node.js
+- git
+
```

Comment:
- [ **OPTIONAL** ] What do you think if you put the  `Prerequisites` section before the command to get a local copy as it contains information needed before the user setup the repo locally? 💪